eclass/mate-desktop.org.eclass

```diff
@@ -50,15 +50,11 @@ inherit versionator
 
 # Set SRC_URI or EGIT_REPO_URI based on whether live
 if [[ ${PV} == 9999 ]]; then
-	EGIT_REPO_URI="
-		https://github.com/mate-desktop/${MATE_DESKTOP_ORG_PN}.git
-		git://github.com/mate-desktop/${MATE_DESKTOP_ORG_PN}.git
-		http://github.com/mate-desktop/${MATE_DESKTOP_ORG_PN}.git
-	"
+	EGIT_REPO_URI="https://github.com/mate-desktop/${MATE_DESKTOP_ORG_PN}.git"
```

Comment:
If a MITM attacker can just block TLS connections, they could always force Gentoo users to pull down code that may have been tampered with by an unsafe http:// or git:// connection.